lib/cdo/crowdin/client_extentions.rb

```diff
@@ -13,15 +13,21 @@ module ClientExtensions
       'codeorg-markdown' => 314545,
       'codeorg-restricted' => 464582,
       'codeorg-testing' => 346087,
-      'codeorg-markdown-testing' => 547997
+      'codeorg-markdown-testing' => 547997,
+      'hour-of-code-test' => 634038,
+      'codeorg-restricted-test' => 634320
```

Comment:
This is not necessary, but the lookup values are still used by the translations-monitoring script.